src/lib/operations/create-bundle-and-sign/createBundleAndSign.ts

```diff
@@ -0,0 +1,46 @@
+import { sign } from "../sign/sign";
+import { createData, Signer, Tag } from "arbundles";
+import base64url from "base64url";
+import { stringToBuffer } from "../../utils/arweave/arweaveUtils";
+import { OthentError, OthentErrorID } from "../../server/errors/errors.utils";
+
+export async function createBundleAndSign(
```

Comment:
This doesn't seem to be used in the SDK as it just creates the bundle on the client and calls the `/sign` endpoint through the `Signer`, so this endpoint/function can probably be removed.